system/settings/rbp.xml

```diff
@@ -95,6 +95,7 @@
       </group>
       <group id="3">
         <setting id="audiooutput.ac3transcode" help="37024">
+          <default>false</default>
         </setting>
```

Comment:
@popcornmix with enabling transcoding by default in the main settings.xml I've set it to be disabled in the appliance xml's for both RPi & RPi2, just wanted to confirm this is needed assuming the change for the setting.xml is accepted.